libs/contact/testing/src/index.ts

```diff
@@ -1 +1,2 @@
-export {}
\ No newline at end of file
+export { DaffTestingContactService } from './drivers/testing/contact.service';
+export { DaffInMemoryContactService } from './drivers/in-memory/contact.service';
```

Comment:
These also shouldn't be exported. There should be a module for each of them that provides them via the corresponding token. Like
```
@NgModule({
  imports: [
    CommonModule
  ]
})
export class DaffContactTestingDriverModule {
  static forRoot(): ModuleWithProviders {
    return {
      ngModule: DaffContactTestingDriverModule,
      providers: [
        {
          provide: DaffContactDriver,
          useExisting: DaffTestingContactService
        }
      ]
    };
  }
}
```
Or something like that.

Reply:
So I believe the testing service still needs to be exported, as its purpose is to test different aspects in the base `daffodil/contact` library.

Reply:
This isn't correct. you should be importing the `DaffContactTestingDriverModule` in the relevant test.

Reply:
Okay I fixed the `effects.spec` that was affected by this change in cd4422f5ca68cd864ae0fb22902b7a88e7ef84cc. Let me know y'alls thoughts because I had a surprisingly tricky time getting it to work without the exported service.

Reply:
All other daffodil packages are testing using an exported testing service so this also may be something we should look into for other packages.